### PR TITLE
Editstyle

### PR DESCRIPTION
**Pull request by Gerrit-Sch**

### CODE REVIEW

File: src/components/Navigation.stories.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import Navigation from "./Navigation";
+
+export default {
+  title: "components/Navigation",
+  component: Navigation,
+};
+const Template = (args) => <Navigation {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {};
```
Comment (Zaid0307) [inline src/components/Navigation.stories.js:10]: Sry i dont know how to comment the img, i think you uploaded this picture 2 times src/img/joel2.jpg and src/img/joel.jpg but still the Same picture

**Review by Zaid0307 — COMMENTED** [review]
Nice one

File: src/components/Navigation.stories.js (hunk 1 above)
Comment (Gerrit-Sch) [reply src/components/Navigation.stories.js:10]: Thanks Zaid!! :)

Comment (Zaid0307) [reply src/components/Navigation.stories.js:10]: 😃 you welcome

**Pull request merged**